.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Ansible Lint
+
+on: [push, pull_request]
+
+jobs:
+  ansible-lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Lint Ansible Playbook
+      uses: ansible/ansible-lint-action@master
+      with:
+        targets: |
+          ./
+        override-deps: |
+          ansible-core==2.12
+          ansible-lint==5.3.0
```

Comment:
Use ansible-lint 5.3.2 which was just released to fix some issues

Reply:
I checked and can confirm that the role is ansible-lint clean also with 5.3.2.